src/main/java/Display.java

```diff
@@ -32,33 +30,31 @@ public void goodbye() {
         cli.present(Message.closeApp());
     }
 
-    public void formatOpportunities(Map<Integer, ArrayList> opportunitiesData) {
-        StringBuilder opportunities = new StringBuilder();
-        Map<Integer,ArrayList> sortedMap = new TreeMap<>(opportunitiesData);
+    public void formatOpportunities(List<Opportunity> opportunities) {
```

Comment:
Looks like this method does more than format opportunities. Based on the name I would expect it to return a string of formatted opportunities rather than returning `void`